fixtures/etc/fos4PL8.json

```
@@ -0,0 +1,368 @@
+          "dmxRange": [10, 19],
+          "type": "Generic",
+          "comment": "Linear"
+        },
+        {
+          "dmxRange": [20, 29],
+          "type": "Generic",
+          "comment": "Incandescent Red Shift"
+        },
+        {
+          "dmxRange": [30, 39],
+          "type": "Generic",
+          "comment": "Linear Red Shift"
+        },
+        {
+          "dmxRange": [40, 255],
+          "type": "NoFunction",
+          "comment": "Reserved"
+        }
+      ]
+    },
+    "Fan": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "Maintenance",
+          "comment": "Auto"
+        },
+        {
+          "dmxRange": [10, 19],
+          "type": "Maintenance",
+          "comment": "Off"
+        },
+        {
+          "dmxRange": [20, 20],
+          "type": "Maintenance",
+          "comment": "Slow"
+        },
+        {
+          "dmxRange": [21, 248],
+          "type": "Maintenance",
+          "comment": "Linear Increase In Speed"
+        },
+        {
+          "dmxRange": [249, 249],
+          "type": "Maintenance",
+          "comment": "Fast"
+        },
+        {
+          "dmxRange": [250, 255],
+          "type": "Maintenance",
+          "comment": "Auto"
+        }
+      ]
+    },
+    "Mix": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Generic",
+          "comment": "Full Studio"
+        },
+        {
+          "dmxRange": [1, 254],
+          "type": "Generic",
+          "comment": "Linear interpolation from Studio to RGB"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "Generic",
+          "comment": "Full RGB"
+        }
+      ]
+    },
+    "Color Temperature": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "ColorTemperature",
+          "colorTemperature": "3200K"
+        },
+        {
+          "dmxRange": [1, 165],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "1900K",
+          "colorTemperatureEnd": "6000K"
+        },
+        {
+          "dmxRange": [166, 254],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "6050K",
+          "colorTemperatureEnd": "10450K"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "ColorTemperature",
+          "colorTemperature": "5600K"
+        }
+      ]
+    },
+    "Tint": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Generic",
+          "comment": "Neutral"
+        },
+        {
+          "dmxRange": [1, 127],
+          "type": "Generic",
+          "comment": "Linear between +10 and 0"
+        },
+        {
+          "dmxRange": [128, 128],
+          "type": "Generic",
+          "comment": "Neutral"
+        },
+        {
+          "dmxRange": [129, 255],
+          "type": "Generic",
+          "comment": "Linear between 0 and -10"
+        }
```

Again, please use `parameter` or `parameterStart`/`parameterEnd` instead of `comment`:

```suggestion
        {
          "dmxRange": [0, 0],
          "type": "Maintenance",
          "parameter": 0,
          "comment": "Neutral"
        },
        {
          "dmxRange": [1, 127],
          "type": "Maintenance",
          "parameterStart": 10,
          "parameterEnd": 0
        },
        {
          "dmxRange": [128, 128],
          "type": "Maintenance",
          "parameter": 0,
          "comment": "Neutral"
        },
        {
          "dmxRange": [129, 255],
          "type": "Generic",
          "type": "Maintenance",
          "parameterStart": 0,
          "parameterEnd": -10
        }
```